css/80_app_fb.css

```diff
@@ -472,4 +472,31 @@ g.layer-ai-features.hide-rapid-buildings path.building  {
 
 .modal-section.rapid-checkbox.poweruser:hover div#tag-sources { 
     background-image: url('img/tag-sources.gif'); 
+}
+
+ul.layer-list.layer-background-list li button.background-favorite-button{
```

Comment:
format nit- add a space between the .css rule specifiers and the opening brace ⬇️ 

`ul.layer-list.layer-background-list li button.background-favorite-button {`